internal/server/boltdbstate/project.go

```diff
@@ -256,12 +256,16 @@ func (s *State) ProjectDelete(ctx context.Context, ref *pb.Ref_Project) error {
 	return nil
 }
 
+func (s *State) ProjectCount(ctx context.Context) (count uint64, err error) {
```

Comment:
Not a big deal, but we generally don't use named returns in our code base.

Reply:
Ah, good to know. I'll remove the named returns.